app/assets/stylesheets/components/_homecards.scss

```diff
@@ -0,0 +1,64 @@
+@import
+url("https://fonts.googleapis.com/css?family=Roboto:400,700");
+body {
```

Comment:
We will decide on a font soon.  Just remember to remove this when we set the font for the whole site.